src/sklearn_evaluation/plot/grid_search.py

```diff
@@ -202,12 +201,17 @@ def _grid_search_single(grid_scores, change, subset, kind, ax, sort, params):
     ax.set_title("Grid search results")
     ax.set_ylabel("Mean score")
     ax.set_xlabel(change)
-    ax.legend(loc="best")
+    ax.legend()
```

Comment:
Im guessing this was removed since best is the default value?

Reply:
yes